### ISSUE TITLE
Encourage files in hosted workflows to be renamed

### ISSUE THREAD

**Issue by garyluu** [dockstore/dockstore#3419]
**Is your feature request related to a problem? Please describe.**
Hosted Workflows should somehow encourage people to change the name of the files.

**Describe the solution you'd like**
Maybe make the default name "untitled.cwl", etc

**Describe alternatives you've considered**
Make the default in the frontend ".cwl", which won't allow the user to save version at all until it's changed

**Additional context**
https://github.com/dockstore/dockstore/pull/3417#discussion_r413225789

┆Issue is synchronized with this [Jira Story](https://ucsc-cgl.atlassian.net/browse/DOCK-1328)
┆Issue Type: Story
┆Sprint: Sprint 34 Isopods
┆Issue Number: DOCK-1328

**Comment by NatalieEO** [comment]
I don't think you can even change the name of the main descriptor for hosted workflows. Or if you can, I don't understand how.

**Comment by garyluu** [comment]
Barring major changes, it would be changing it before they manage they can save.

**Comment by NatalieEO** [comment]
After some discussion in storytime, the priority of this ticket will be bumped out of 1.9.0. 
Discussion summary:
- The backend and UI depend on the primary descriptor being named `Dockstore.x`
- It is a good feature to have. Users should be able to name the primary descriptor what they want, like they can for remote entries.
- You can create a hosted tool version with both a wdl and cwl primary descriptor file
- This ticket might be made easier if we don't support a single tool version to have both a cwl and wdl.
- Currently we do have hosted tools by users other than our team members, but should check whether they make use of a version having both descriptor types before dropping support.
